src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
```

Comment:
health is the same for all examples of class
u can do not add it to args and just wrote like this

`this.health = 100;`